storage-operator/pkg/apis/storage/v1alpha1/volume_types.go

```diff
@@ -103,20 +101,19 @@ type VolumeCondition struct {
 	// Last time the condition transited from one status to another (optional).
 	LastTransitionTime metav1.Time `json:"lastTransitionTime,omitempty"`
 	// Unique, one-word, CamelCase reason for the condition's last transition.
+	// +kubebuilder:validation:Enum=Pending;Terminating;InternalError;CreationError;DestructionError;UnavailableError
 	Reason ConditionReason `json:"reason,omitempty"`
 	// Human readable message indicating details about last transition.
 	Message string `json:"message,omitempty"`
 }
 
 // VolumeStatus defines the observed state of Volume
-// +k8s:openapi-gen=true
 type VolumeStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
 
 	// List of conditions through which the Volume has or has not passed.
-	// +kubebuilder:validation:Enum=Available,Pending,Failed,Terminating
```

Comment:
I would almost fix that in a dedicated commit, since it's a bugfix and not exactly related to the upgrade per se.